irb.gemspec

```diff
@@ -16,6 +16,11 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/ruby/irb"
   spec.licenses      = ["Ruby", "BSD-2-Clause"]
 
+  spec.metadata["homepage_uri"] = spec.homepage
+  spec.metadata["source_code_uri"] = spec.homepage
+  spec.metadata["documentation_uri"] = spec.homepage
```

Comment:
I specifically override the documentation's destination because the readme here has the most up-to-date info. We can modify it later when we improve the documentation on docs.ruby-lang.org.